hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -20,7 +20,7 @@
 <jboss-deployment-structure>
   <deployment>
     <dependencies>
-      <module name="org.hawkular.bus" />
+      <module name="deployment.hawkular-alerts-rest.war" />
```

Comment:
Yes. We need to touch a couple of things for .ear.
One are dependencies, and second is to extract/parametrize the JNDI location to include the .ear.
As we need to coordinate, I put that on my todo list to do it once the .ear is consolidate.
This PR is just to work with the present upstream status.